subgraphs/osmosis/src/common/initializer.ts

```diff
@@ -0,0 +1,414 @@
+import { Protobuf } from "as-proto";
+import { BigDecimal, BigInt, cosmos, log } from "@graphprotocol/graph-ts";
+import {
+  Account,
+  Token,
+  _TokenPrice,
+  DexAmmProtocol,
+  FinancialsDailySnapshot,
+  LiquidityPoolDailySnapshot,
+  LiquidityPoolHourlySnapshot,
+  UsageMetricsDailySnapshot,
+  UsageMetricsHourlySnapshot,
+  LiquidityPool as LiquidityPoolStore,
+  LiquidityPoolFee,
+} from "../../generated/schema";
+import * as constants from "../common/constants";
+import * as utils from "../common/utils";
+import { MsgCreateBalancerPool } from "../modules/Decoder";
+import { initRegistry } from "./registry";
+
+export function getOrCreateAccount(id: string): Account {
+  let account = Account.load(id);
+
+  if (account == null) {
+    account = new Account(id);
+    account.save();
+
+    const protocol = getOrCreateDexAmmProtocol();
+    protocol.cumulativeUniqueUsers += 1;
+    protocol.save();
+  }
+
+  return account;
+}
+
+export function getOrCreateLiquidityPoolFee(
+  feeId: string,
+  feeType: string,
+  feePercentage: BigDecimal = constants.BIGDECIMAL_ZERO
+): LiquidityPoolFee {
+  let fees = LiquidityPoolFee.load(feeId);
+
+  if (fees == null) {
+    fees = new LiquidityPoolFee(feeId);
+    fees.feeType = feeType;
+    fees.feePercentage = feePercentage;
+
+    fees.save();
+  }
+
+  if (feePercentage.notEqual(constants.BIGDECIMAL_ZERO)) {
+    fees.feePercentage = feePercentage;
+    fees.save();
+  }
+
+  return fees;
+}
+
+export function getOrCreateDexAmmProtocol(): DexAmmProtocol {
+  const protocolId = constants.Protocol.NAME;
+  let protocol = DexAmmProtocol.load(protocolId);
+
+  if (protocol == null) {
+    protocol = new DexAmmProtocol(protocolId);
+    protocol.name = constants.Protocol.NAME;
+    protocol.slug = constants.Protocol.SLUG;
+    protocol.network = constants.Network.OSMOSIS;
+    protocol.type = constants.ProtocolType.EXCHANGE;
+
+    //////// Quantitative Data ////////
+    protocol.totalValueLockedUSD = constants.BIGDECIMAL_ZERO;
+    protocol.cumulativeSupplySideRevenueUSD = constants.BIGDECIMAL_ZERO;
+    protocol.cumulativeProtocolSideRevenueUSD = constants.BIGDECIMAL_ZERO;
+    protocol.cumulativeTotalRevenueUSD = constants.BIGDECIMAL_ZERO;
+    protocol.cumulativeVolumeUSD = constants.BIGDECIMAL_ZERO;
+    protocol.protocolControlledValueUSD = constants.BIGDECIMAL_ZERO;
+    protocol.cumulativeUniqueUsers = 0;
+    protocol.totalPoolCount = 0;
+
+    protocol.schemaVersion = constants.Protocol.SCHEMA_VERSION;
+    protocol.subgraphVersion = constants.Protocol.SUBGRAPH_VERSION;

```

Comment:
Will you update this to get the versions from the Version.ts file that gets generated with use of the CLI? This ensures that the versions in the subgraph are always up to date with the deployment.json